Add dev container and updated dev documentation with other cleanups (#2)
LibrePhotos/librephotos.docs: _posts/Installation/0000-01-03-ubuntu_debian_install.md

---
title: "Manual .deb installation"
excerpt: "Coming Soon"
last_modified_at: 2021-3-06
category: 1 
---
[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=LibrePhotos_ownphotos&metric=alert_status)](https://sonarcloud.io/dashboard?id=LibrePhotos_ownphotos) ![Discord](https://img.shields.io/discord/784619049208250388?style=plastic) ![Website](https://img.shields.io/website?down_color=lightgrey&down_message=offline&style=plastic&up_color=blue&up_message=online&url=https%3A%2F%2Flibrephotos.com) ![GitHub contributors](https://img.shields.io/github/contributors/librephotos/librephotos?style=plastic)

> > Currently the project is in very early stages, some bugs may exist. If you find any please log an issue

Reviewer: ```suggestion
> Currently the project is in very early stages; some bugs may exist. If you find any please log an issue
```
Grammar

## Overview

> This page will hold how to install LibrePhotos via .deb. This method is currently being worked on and the how to will be published once in beta.

Reviewer: ```suggestion
This page will hold how to install LibrePhotos as a .deb file. This method is currently being worked on. and this guide will be published once it is in beta.
```
Readability
